src/Microsoft.ML.Parquet/ParquetLoader.cs

```diff
@@ -445,8 +445,8 @@ private sealed class Cursor : RootCursorBase
             private int _curDataSetRow;
             private IEnumerator<int> _dataSetEnumerator;
             private IEnumerator<int> _blockEnumerator;
-            private IList[] _columnValues;
-            private Random _rand;
+            private readonly IList[] _columnValues;
+            private readonly Random _rand;
```

Comment:
The changes to `readonly` here at least seem positive.

Reply:
The ones in: Microsoft.ML.Data/Transforms/ValueMapping.cs seem ok to keep too.